### PR TITLE
[medVtkViewItkDataImage4DInteractor]: fix wrong frameRateComputation

### PR DESCRIPTION
**Pull request by paulineMig**
fix https://github.com/medInria/medInria-public/issues/850

### CODE REVIEW

**Review by mathildemerle — APPROVED** [review]

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp [hunk 1]
```diff
@@ -122,11 +122,12 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
 
         if (SetViewInput(data, layer) )
         {
+            double frameRate = static_cast<double>(m_poConv->getNumberOfVolumes()) / static_cast<double>(m_poConv->getTotalTime());
```
Comment (fcollot) [inline src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp:125]: Just for info, there is no need to cast both operands to double. Only one is sufficient (a division involving a double results in a double).

**Review by fcollot — APPROVED** [review]

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp (hunk 1 above)
Comment (paulineMig) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp:125]: True i just did a copy paste without thinking do you want me to modify it?

Comment (fcollot) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp:125]: no don't worry it's just something I see often in the code so I wanted to point it out to

**Review by mjuhoor — APPROVED** [review]

File: src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp [hunk 2]
```diff
@@ -122,11 +122,12 @@ void medVtkViewItkDataImage4DInteractor::setInputData(medAbstractData *data)
...
             d->imageData->setMetaData("SequenceDuration", QString::number(m_poConv->getTotalTime()));
-            d->imageData->setMetaData("SequenceFrameRate", QString::number((double)(m_poConv->getNumberOfVolumes() -1 )/ (double)m_poConv->getTotalTime()));
```
Comment (nebatmusic) [inline src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp:126]: What is the frame rate. If it is the number of volumes per unit of time, you should keep it as it was.

Comment (mathildemerle) [reply src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImage4DInteractor.cpp:126]: framerate is now computed line 125 with this same computation except the "-1"

**Pull request merged**